src/config-map-task.ts

```diff
@@ -10,7 +10,8 @@ import { ConfigMap } from './config-map';
 async function run() {
   let version = task.getInput('version');
   let agentOS = task.osType();
-  let ocPath = await InstallHandler.installOc(version, agentOS);
+  const useLocalOc: boolean = task.getBoolInput('useLocalOc');
```

Comment:
Yes it's the same parameter used in the oc-cmd task. Even in this task the user is going execute an oc command so we still give him the chance to choose if he wants to use a local version of oc or download it. Also in this case if the user don't check it, it will work as before, so let's say it's already optional.